ur_moveit_config/launch/test_moveit_servo_velocity_controller.launch.py

```diff
@@ -260,7 +271,10 @@ def generate_launch_description():
 
     kinematics_yaml = load_yaml("ur_moveit_config", "config/kinematics.yaml")
 
-    joint_limits_yaml = {"robot_description_planning": joint_limit_params}
+    joint_limits_yaml = load_yaml(
```

Comment:
Same comment about not wanting to duplicate the loading of joint_limits.yaml here.

Reply:
7c862d6